stable/cluster-autoscaler/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 0.7.0
+version: 0.7.1
```

Comment:
Why don't you update the appVersion to 1.12.0?

Reply:
I did not believe it was the purpose of this MR but I'm happy to add it if you want.

Reply:
Currently, the PR adds a requirement for `v1.12.0` but doesn't use `v1.12.0`.

Reply:
sounds good, sorted 👍